client/my-sites/posts/post.jsx

```diff
@@ -278,7 +278,7 @@ module.exports = React.createClass({
 
 		if ( showLikes ) {
 			if ( post.like_count > 0 ) {
-				likeTitle = this.translate( '1 Like', '%(count)s Likes', {
+				likeTitle = this.translate( '%(count)s Like', '%(count)s Likes', {
```

Comment:
No, this is not @jordwest.
The changes that remove those warnings are made in `posts-navigation.jsx` file.